CHANGES.rst

```diff
@@ -12,6 +12,7 @@ Bug fix:
  * Xcode11: Show AuthenticationViewController fullscreen (PR #3093).
  * Xcode11: Fix font used for `org.matrix.custom.html`messages in timeline (#3241).
  * Settings: New phone number is invisible in dark theme (#3218).
+ * Xcode11: Fix decryption on notifications when the key is not present (PR #3297).
```

Comment:
```suggestion
 * Xcode11: Fix decryption on notifications when the key is not present (#3295).
```

We usually refer to the issue instead of the PR